net.adamec.lib.common.dmn.engine/engine/definition/DmnVariableDefinition.cs

```diff
@@ -140,10 +140,10 @@ public static string NormalizeVariableName(string name)
             if (string.IsNullOrWhiteSpace(retVal))
                 throw new ArgumentException($"Variable name is null or empty",nameof(name));
 
-            if (retVal.Any(c => !(c >= 'a' && c <= 'z' || c >= 'A' && c <= 'Z' || c >= '0' && c <= '9' || c == '_')))
+            if (retVal.Any(c => char.IsLetter(c) || char.IsDigit(c)  || c == '_')))
```

Comment:
I guess the line should be following (missing opening bracket and not operator) 
```
if (retVal.Any(c => !(char.IsLetter(c) || char.IsDigit(c)  || c == '_')))
```

Reply:
absolutly, corrected the statement.